docs/operations/metrics.md

```diff
@@ -258,6 +258,8 @@ batch ingestion emit the following metrics. These metrics are deltas for each em
 |`ingest/notices/time`|Milliseconds taken to process a notice by the supervisor.|`dataSource`, `tags`| < 1s |
 |`ingest/pause/time`|Milliseconds spent by a task in a paused state without ingesting.|`dataSource`, `taskId`, `tags`| < 10 seconds|
 |`ingest/handoff/time`|Total number of milliseconds taken to handoff a set of segments.|`dataSource`, `taskId`, `taskType`, `groupId`, `tags`|Depends on the coordinator cycle time.|
+|`ingest/skipScaleUp/count`|Total number of times `lagBased` auto scaler attempted to scale up but was skipped due to `minTriggerScaleActionFrequencyMillis`.|`dataSource`, `stream`, `tags`|Depends on auto scaler config.|
```

Comment:
How about we just emit the task count as computed by the auto scaler?
Something like `task/autoScaler/requiredTasks/count`.
If this value is found to be different from the current task count, we can include an optional dimension `scaleSkipReason`.

I feel this metric would be more beneficial than just tracking when scaling actions were skipped. It would help us:
- See how the scaler behaves and computes required task count at different stages
- Know the exact gap between current and desired counts
- Know the reason scaling was skipped, if so

What do you think @adithyachakilam , @abhishekrb19 ?

PS: If required, the current task count may also be emitted as a separate metric but I think `task/running/count` itself should suffice.

Reply:
I like this idea 👍

Reply:
Makes sense, restructured the PR.